internal/backend/triggersgrpcsvc/svc.go

```diff
@@ -126,9 +126,15 @@ func (s *server) List(ctx context.Context, req *connect.Request[triggersv1.ListR
 		return nil, sdkerrors.AsConnectError(err)
 	}
 
+	pid, err := sdktypes.ParseProjectID(msg.ProjectId)
+	if err != nil {
+		return nil, sdkerrors.AsConnectError(err)
+	}
+
 	filter := sdkservices.ListTriggersFilter{
 		EnvID:        eid,
 		ConnectionID: cid,
+		ProjectID:    pid,
```

Comment:
why do you need projectID if you have envID ?

Reply:
this was a bug, mate. We have filter with projectID, but we are not passing projectID

Reply:
so when planning the manifest, we were deleting triggers for other projects.
Why? because we failed to list triggers for THIS project to plan

Reply:
so it's additional option to filter  ?